### PR TITLE
Add task to execute commands over ssh and allow customization of template for nebula-config generation 

### PR DESCRIPTION
**Pull request by kimpaller**
Implements:

1. Add task to execute commands over ssh.
2. Allow use of custom nebula-config template for nebula-config generation.

### CODE REVIEW

File: nebula/tasks.py [hunk 1]
```diff
@@ -458,10 +458,27 @@ def update_boot_files_manager(
...
+    },
+)
+def initialize_manager(
+    c,
+    yamlfilename="/etc/default/nebula",
+    board_name=None,
+):
+    """Initialize manager class. Specially useful for diagnostic purposes"""
+    m = nebula.manager(configfilename=yamlfilename, board_name=board_name)
+    m.get_status()
```
Comment (tfcollins) [inline nebula/tasks.py:474]: This suppose to be a blank method? What is this suppose to do?

Comment (kimpaller) [reply nebula/tasks.py:474]: The goal is  to check if defined connections (i.e JTAG, UART)  are indeed available. This is done by initializing manager class, as it has already implemented the necessary checks, but i guess this modifications is not necessary anymore as @tagoylo have already implemented the tasks to do this on https://github.com/sdgtt/nebula/pull/78, will just remove this commit.

File: nebula/tasks.py [hunk 2]
```diff
@@ -863,11 +880,40 @@ def run_diagnostics(
...
+    c,
+    ip=None,
+    user="root",
+    password="analog",
+    board_name=None,
+    command=None,
+    ignore_exception=False,
+    retries=3,
+):
+    """Run command to expand sd card via IP"""
```
[the remark this answers is not in the document]
Comment (kimpaller) [reply nebula/tasks.py]: fixed in 80d3db1

File: nebula/tasks.py (hunk 1 above)
Comment (kimpaller) [reply nebula/tasks.py]: Code removed.

**Review by tagoylo — APPROVED** [review]

**Pull request merged**